client/boot/common.js

```diff
@@ -241,6 +242,18 @@ function setupErrorLogger( reduxStore ) {
 		return;
 	}
 
+	Sentry.init( {
+		dsn: 'https://61275d63a504465ab315245f1a379dab@o248881.ingest.sentry.io/6313676',
+		environment: config( 'env_id' ),
+		initialScope: {
+			user: { id: getCurrentUserId( reduxStore.getState() ) },
+		},
+	} );
+	if ( window._jsErr ) {
+		window._jsErr.forEach( ( error ) => Sentry.captureException( error ) );
+		Sentry.flush().then( () => delete window._jsErr );
```

Comment:
Copied this from the existing ETK solution, but I'm not sure it makes sense to delete the error array -- unless we delete the event listener as well, the array would just be recreated.